### ISSUE TITLE
Is it possible to use github releases to host prebuilt binaries again?

### ISSUE THREAD

**Issue by gemwuu** [balena-io-modules/drivelist#419]
It's almost 5 years since the last release on GitHub 🤔️.
![image](https://user-images.githubusercontent.com/4635838/212802056-d5931105-4556-4d13-a15e-3dd6a9916e8f.png)

**Comment by petervanderwalt** [comment]
Agreed - we run into rebuild issues far to often - would be so great to have prebuilds again

**Comment by lurch** [comment]
See also #331
